xml/admin_administration.xml

```diff
@@ -75,13 +77,30 @@
   </note>
 
   <sect2 xml:id="transactional.updates.installation">
-   <title>Manual Installation of Updates</title>
+   <title>Manually Applying Updates</title>
+   
    <para>
     After the <command>transactional-update</command> script has run on all
     nodes, &dashboard; displays any nodes in your cluster running outdated
-    software. &dashboard; then enables you to update your cluster directly.
+    software. The updates are only applied after a reboot. For this purpose,
+    &dashboard; enables you to update your cluster directly.
     Follow the next procedure to update your cluster.
    </para>
```

Comment:
What about adding something like: Note that the cluster update via velum might also change configuration settings due to the new software. IF you reboot a machine without doing the orchestration via velum, the configuration update will not happen and this might break your cluster.

Thorsten, @flavio , is this correct?

Reply:
This is also covered in the section I've added, rewritten.

Reply:
@ajaeger yes, that's correct